devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java

```diff
@@ -28,6 +35,57 @@ private CreateUtils() {
         //Not to be constructed
     }
 
+    private static boolean isVersionRange(String versionStr) {
+        if (versionStr == null || versionStr.isEmpty()) {
+            return false;
+        }
+        char c = versionStr.charAt(0);
+        if (c == '[' || c == '(') {
+            return true;
+        }
+        c = versionStr.charAt(versionStr.length() - 1);
+        if (c == ']' || c == ')') {
+            return true;
+        }
+        return versionStr.indexOf(',') >= 0;
+    }
+
+    private static String nonEmptyOrNull(String value) {
```

Comment:
Reuse `org.apache.commons.lang3.StringUtils` instead

Reply:
Is there any other benefit? I kind of don't like to add a new dependency on the whole library just for one trivial method.

Reply:
You'll not add a new dependency, since this class already uses `StringUtils`  (see the imports)

Reply:
Ah, you're right, sorry, will change then.

Reply:
Done.

Reply:
The suggestion was to remove the method completely :) See https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotEmpty-java.lang.CharSequence-

Reply:
Actually the method in this case is: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#defaultIfBlank-T-T-

Reply:
Done. Thanks.